src/MPDevice.cpp

```diff
@@ -6833,14 +6833,14 @@ void MPDevice::importFromCSV(const QJsonArray &creds, const MPDeviceProgressCb &
         QJsonObject qjobject = creds[i].toObject();
 
         /* Check login size */
-        if (qjobject["login"].toString().length() >= pMesProt->getLoginMaxLength()-1)
```

Comment:
what's the value for the mini & the mini ble?

Reply:
BLE: 64
Mini: 63